### PR TITLE
Portfolio Mobile-First-finished

### PR DESCRIPTION
**Pull request by Ramaunspoken**
Changes and Modification
Created a work section and About section alining them with flexbox

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -92,3 +102,161 @@ a:hover {
...
+  background: #396df2;
+}
+
+/* Css styles for about section */
+
+.about {
+  display: grid;
+  row-gap: 15px;
+  padding: 0 15px;
+}
```
Comment (emmiiorji) [inline style.css:667]: Your design looks great. Kindly ensure that you use the Poppins font family and the same font color as used by the template. This also applies to other sections of the website.

Stick to the design and the result will be great.

Your design:
image

Figma design:
image

File: style.css [hunk 2]
```diff
@@ -92,3 +102,161 @@ a:hover {
...
+  padding: 10px 20px;
+  background: #fff;
+  border-radius: 10px;
+  border: 1px solid #396df2;
+}
+
+.default_btn:hover {
+  color: #fff;
+  background: #396df2;
+}
```
Comment (emmiiorji) [inline style.css:649]: You did a good job at implementing the hover button interaction. Kindly look into implementing the remaining interactions like the button pressed interaction.

Button interactions given by design:
image

File: index.html [hunk 3]
```diff
@@ -73,6 +76,198 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
...
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/Linkedin icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
```
Comment (emmiiorji) [inline index.html:220]: Kindly ensure that the social icons are horizontally aligned as shown by the design.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Ramaunspoken, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more

## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow
- Repo looks professional


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -73,6 +76,198 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
           </ul>
         </div>
       </section>
```
Comment (HAMMAS-SALEEM) [inline index.html]: Capture

**[Optional]**

- [ ] Your background is not expanding on the whole screen as you can see. Kindly fix it to make your design look similar to the Figma template 😊to make your design look professional 👍

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @Ramaunspoken :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Github flow
:heavy_check_mark: Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Ramaunspoken) [reply index.html]: Thanks i will #Hammas-saleem

**Comment by Ramaunspoken** [comment]
Issues resolved

**Pull request closed without merging**